features/step_definitions/brief_response_steps.rb

```diff
@@ -24,7 +24,8 @@
   puts "brief id: #{@brief['id']}"
   @lot_slug = @brief['lotSlug']
   @framework_slug = @brief['frameworkSlug']
-  @buyer_user = (@brief['users'].select { |u| u["active"] && !u["locked"] })[0]
+  @buyer_user = @brief['users'].sample
+  ensure_user_can_log_in @buyer_user
```

Comment:
Would we have to deactivate the user again afterwards - some of our scenarios test that inactive users can't do things?

Reply:
I didn't think it would be an issue. The only test I could find where we do this we create an inactive user; I think we don't even have a way (in our functional tests) of getting an existing inactive user.